src/compiler/emitter.ts

```diff
@@ -2584,6 +2614,217 @@ namespace ts {
             emitInitializer(node.initializer, node.name.end, node);
         }
 
+        //
+        // JSDoc
+        //
+        function emitJSDocComment(node: JSDoc) {
+            write("/**");
+            if (node.comment) {

```

Comment:
This is lumping the empty string and an `undefined` comment into the same behavioral bucket. Intentional?